.nvmrc

```
@@ -1 +1 @@
-v18
+v20
```

```suggestion
lts/*
```

To match the module template.